extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/CancelTask.java

```diff
@@ -0,0 +1,31 @@
+package io.fabric8.tekton.api.examples;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Component;
```

Comment:
Error:  /home/runner/work/kubernetes-client/kubernetes-client/extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/CancelTask.java:[5,38] package org.springframework.stereotype does not exist

Reply:
Cleaned up.